idea/idea-frontend-independent/src/org/jetbrains/kotlin/idea/quickfix/RemoveModifierFix.kt

```diff
@@ -61,6 +61,7 @@ class RemoveModifierFix(
                 isRedundant = true
             )
         val removeOpenModifier = createRemoveModifierFromListOwnerPsiBasedFactory(KtTokens.OPEN_KEYWORD)
+        val removePrivateModifier: QuickFixesPsiBasedFactory<PsiElement> = createRemoveModifierFromListOwnerPsiBasedFactory(KtTokens.PRIVATE_KEYWORD)
```

Comment:
Nit: explicit type for `val` seems not necessary (as the similar one above didn't specify that).

Reply:
Let me change the line above to specify it so it's consistent with the ones above.

Reply:
Oh, everything except the above one specified the type, cool!